code/script.js

```diff
@@ -0,0 +1,92 @@
+const USER = `Ajliin`
+const REPOS_URL = `https://api.github.com/users/${USER}/repos`
+
+const firstContainer = document.getElementById('first')
+const projectContainer = document.getElementById('projects')
+
+// ------------FIRST FETCH FUNCTION------------
+const fetchRepos =() => {
+fetch(REPOS_URL)
+    .then ((response) => {
+        return response.json()
+    })
+    .then ((data) => {
+        repoInfo(data)   
+    })
+}
+fetchRepos()
+
+// ------------function for first lines inside FIRST FETCH FUNCTION------------
+
+const repoInfo = ((data) =>{
+    first.innerHTML = `
+    <div>
+        <h1>Welcome to ${data[0].owner.login}s GitHub Tracker</h1>
+        <img class='profile-img' src='https://avatars.githubusercontent.com/u/84288114?v=4' alt='profile pic Ajliin'>
+    </div>
+    <div class=info-square>
+        <h2>Elin Diczfalusy</h2>
+        <p>User name: ${data[0].owner.login}</p>
+        <p>GitHub page: <a href='${data[0].owner.html_url}'>${data[0].owner.html_url}</a></p>
+    </div>`
+
+// ------------function filters out the forked repos from Technigo
+    const forkedRepo = data.filter((repo) => repo.fork && repo.name.startsWith('project-'))
+// ------------function to chart.js on how many repos      
+    drawChart(forkedRepo.length)
+// ------------forEach repo --> send to innerHTML   
```

Comment:
Good job on the amount of comments on your code in general, so easy to follow and understand what everything does! :)